projects/components/src/table/table.component.scss

```diff
@@ -106,7 +107,7 @@
 }
 
 .state-watcher {
-  height: 100%;
+  height: calc(100% - #{$paginator-height});
```

Comment:
curious. what does #{} do?

Reply:
[Interpolation](https://sass-lang.com/documentation/interpolation). Its so we can use the SCSS variable in the CSS function. It works like html templates in typescript.